packages/core/src/http-client/http-client-types.ts

```diff
@@ -74,8 +74,3 @@ export interface HttpResponse extends KnownHttpResponseFields {
 export interface HttpReponse extends KnownHttpResponseFields {
   [otherKey: string]: any;
 }
-
-export interface HttpRequestAndResponse{
```

Comment:
Should we only deprecate? Was this exported to root level so some could use it? If we remove it then add a compatibility node.

Reply:
This interface is still under the canary version. I guess it's fine.

Reply:
Agreed.